pkg/ruler/ruleset.go

```diff
@@ -281,8 +281,9 @@ func GenerateInTotoLink(reports []Report, fileBytes []byte) in_toto.Metablock {
 
 	materials := make(map[string]interface{})
 	request := make(map[string]interface{})
+	fileBytes = append(fileBytes, 10) // FIXME: we need an additional newline here?!
```

Comment:
Is this `FIXME` going to be resolved before merging?

Reply:
I think I made a mistake labelling this as a "fixme". I think it's an issue with the canonical serializer that ends things with a newline on the other implementations. I think I'd like to leave the note for people to know, but maybe drop the fixme?  what do you think?

Reply:
Yeah just drop the fixme and add a bit more of an explanation of why :)